docs/src/app/pages/Spinners/ExampleWithContainerFixed.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Spinner, SpinnerContainer } from 'react-lds';
+
+const ExampleWithContainerFixed = () => (
+  <SpinnerContainer>
+    <Spinner size="medium" />
+  </SpinnerContainer>
+);
+
+// the example is not actually fixed, because it would cover the whole page.
+// I could put it in an iframe to contain the effect, but maybe thats over the top
```

Comment:
@brunottonurb it's alright to document the prop and not add an example. remove this example